integration_tests/fixtures/forseti/modules/policy_library/templates/policy-library/lib/util_test.rego

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+package validator.gcp.lib
+
+# has_field tests
```

Comment:
should not have this comment as it just duplicates the method name without any new info

Reply:
Similar to above, these files are all directly from the policy library. I don't have plans to modify them at all.